_posts/2017/2017-03-29-epages-hackathon-2017.md

```diff
@@ -0,0 +1,67 @@
+---
+layout: post
+title: "ePages Hackathon 2017"
+date: "2017-03-29 06:39:20"
+image: blog-header/blue-sky.jpg
+categories: events
+authors: ["Rose"]
+---
+<style>
+.twitter-tweet {
+  margin: auto;
+}
+</style>
+
+As every year in spring, last week we organized our third hackathon at ePages.
+This year it has been all about trying out new stuff for our very latest backend architecture ePages BEYOND, adding value to ePages Now as well as automating internal processes.
+
+## Getting prepared
+
+Just as last year, our Scrum Masters did an outstanding job organizing our two-day hackathon and ensuring a smooth flow of the whole event.
+They organized travel tickets and hotels rooms for the colleagues from Barcelona, New York, and Jena, checked with our CTO upon final topics and the time schedule, prepared meeting rooms for the teams, and most importantly ordered delicious snacks, drinks, and food {% emoji wink %}.
+
+## And then off we go
+
+The first day started off with a surprisingly blue sky and a vitalizing breakfast to get us fuelled and ready to hack away.
+The atmosphere was very relaxed and we had a lot of time to chat with colleagues we had not seen for some time.
+
+<blockquote class="twitter-tweet" data-lang="en"><p lang="en" dir="ltr">Ready to hack away! Starting off with a breakfast! <a href="https://twitter.com/hashtag/ephack17?src=hash">#ephack17</a> <a href="https://t.co/Gf552CgNc9">pic.twitter.com/Gf552CgNc9</a></p>&mdash; ePages Developers (@epagesdevs) <a href="https://twitter.com/epagesdevs/status/844451431872647168">March 22, 2017</a></blockquote>
+<script async src="//platform.twitter.com/widgets.js" charset="utf-8"></script>
+
+After the breakfast, each of us started to hack away in one of the 13 teams - all with different exciting topics.
```

Comment:
After having seen the video, Christina told me, we had 14 teams> the video team included ;-)